.github/workflows/actions.yml

```diff
@@ -87,7 +87,7 @@ jobs:
     needs: [version-getter, build-macos]
     uses: ./.github/workflows/test_macos.yml
     with:
-      artifact-file: "SuperCollider-${{ needs.version-getter.outputs.sc-version }}-macOS-universal.dmg"
+      artifact-file: "SuperCollider-${{ needs.version-getter.outputs.sc-version }}-macOS-x64.dmg"
```

Comment:
I wouldn't say intel is a legacy platform just yet... but I do get your point.
In short, the practical reason for this is that if we run tests on macOS >12, we encounter #6488

Reply:
Oh right, this one. Then lets keep it this way.

I sadly think intel is a legacy platform (still using one myself) :/ We should focus more towards arm because the quantity of intel mac users will not increase in the future.